src/app/_components/GoogleLogin/GoogleLoginButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button } from "@mui/material";
+import React from "react";
+import GoogleIcon from "@mui/icons-material/Google";
+
+interface GoogleLoginButtonProps {
+  onLogin: () => void;
+}
+
+const GoogleLoginButton = ({
+  onLogin,
+}: GoogleLoginButtonProps): JSX.Element => {
+  const handleLogin = () => {
+    onLogin();
+  };
+
+  return (
+    <Button
```

Comment:
for the width and height as well as the border radius will probably be taken care of by the container this component is being placed into so you probably won't need to hard code this.

Reply:
static sizing are removed. Thanks